pkg/controller/clone/planner_test.go

```diff
@@ -818,22 +816,6 @@ var _ = Describe("Planner test", func() {
 			validateHostClonePhase(planner, args, plan[2])
 			validateRebindPhase(planner, args, plan[3])
 		})
-
-		It("should fail planning host-assisted clone from snapshot when no valid storage class for source PVC is found", func() {
```

Comment:
This is a case that should be tested.  It is totally possible that a snapshot is created, the storage class is deleted, and we cannot create the temp claim